added a issue in opera mini (#41)
riverar/wsa-app-compatibility: README.md

| Office lens | 16.0.14527.20178 | ❌ || Might require GMS, cannot sign in |
| Opera Browser Beta | 65.1.3381.61349 (x86_64) | ✔️ || Change app layout to Tablet Mode for a better experience |
| Opera GX : Gaming Browser | 1.3.6 | ✔️ |||

> | Opera Mini Beta | 61.0.2254.59921 | 🆖 |Dark mode doesn't work since it uses Draw over screen dark overlay which is not supported by WSA| Crashes without GSM (with no warning) |

Reviewer: ```suggestion
| Opera Mini Beta | 61.0.2254.59921 | 🆖 | Dark mode doesn't work due to inability to _draw over other apps_ | Crashes without GSM (with no warning) |
```
